client.go

```diff
@@ -10,6 +11,9 @@ import (
 
 const ROOT = "https://api.twilio.com"
 const VERSION = "2010-04-01"
+const IP_MESSAGING_ROOT = "https://ip-messaging.twilio.com"
```

Comment:
instead of creating a new set of HTTP methods specific to IPMessaging (getIP, etc.), can you make a TwilioIPMessagingClient that implements the ip-messaging.twilio.com subdomain in the `RootUrl` function and aliases everything to the Client? That way, we aren't duplicating a bunch of code. (The original design of this library was before subdomains existed in the Twilio API, and also before people knew what the "Go" way to do things was =P)